ci_framework/roles/libvirt_manager/tasks/virsh_checks.yml

```diff
@@ -34,3 +68,19 @@
     name: "{{ cifmw_libvirt_manager_user }}"
     groups: libvirt
     append: true
+
+- name: Allow QEMU on home directory for the storage access
+  become: true
+  ansible.posix.acl:
+    path: "{{ ansible_user_dir }}"
+    entity: "qemu"
+    etype: "user"
+    permissions: rx
+    state: present
+
+- name: Inject system connection parameters in bashrc
```

Comment:
the most important part in the doc is that actually:
```
If virsh finds the environment variable VIRSH_DEFAULT_CONNECT_URI set, it will try this URI by default. Use of this environment variable is, however, deprecated now that libvirt supports LIBVIRT_DEFAULT_URI itself.
```
VIRSH one is deprecated. I'll remove it.